include/net/xia_ether.h

```diff
@@ -16,6 +15,61 @@
 /* ETHER's virtal XID type. */
 int ether_vxt __read_mostly = -1;
 
+/* ETHER-INTERFACE. */
+struct ether_interface {
+	struct net_device  *dev;
+	atomic_t           refcnt;
+	int                dead;
+	struct rcu_head    rcu_head;
+
+	atomic_t           neigh_cnt;
+	spinlock_t         interface_lock;
+	struct list_head   list_interface_common_addr;
+};
+
+static inline struct ether_interface *__ether_get_rtnl(
+	        const struct net_device *dev)
+{
+	return rtnl_dereference(dev->eth_ptr);
+}
+
+static inline void interface_finish_destroy(struct ether_interface *eint)
```

Comment:
Once the `dead` field is gone, this function will become a two-line function. Given that `interface_finish_destroy()` is only called in `ether_interface_put()`, please merge the body of `interface_finish_destroy()` into `ether_interface_put()`.